src/py/aspen/database/models/sequences.py

```diff
@@ -0,0 +1,145 @@
+    ILLUMINA_MINISEQ = "Illumina MiniSeq"
+    ILLUMINA_MISEQ = "Illumina MiSeq"
+    ILLUMINA_NEXTSEQ_500 = "Illumina NextSeq 500"
+    ILLUMINA_NEXTSEQ_550 = "Illumina NextSeq 550"
+    PACBIO_RS = "PacBio RS"
+    PACBIO_RS_II = "PacBio RS II"
+    PACBIO_SEQUEL = "PacBio Sequel"
+    PACBIO_SEQUEL_II = "PacBio Sequel II"
+    ION_TORRENT_PGM = "Ion Torrent PGM"
+    ION_TORRENT_PROTON = "Ion Torrent Proton"
+    ION_TORRENT_S5_XL = "Ion Torrent S5 XL"
+    ION_TORRENT_S5 = "Ion Torrent S5"
+    OXFORD_NANOPORE_GRIDION = "Oxford Nanopore GridION"
+    OXFORD_NANOPORE_MINION = "Oxford Nanopore MinION"
+    OXFORD_NANOPORE_PROMETHION = "Oxford Nanopore PromethION"
+    BGISEQ_500 = "BGISEQ-500"
+    DNBSEQ_T7 = "DNBSEQ-T7"
+    DNBSEQ_G400 = "DNBSEQ-G400"
+    DNBSEQ_G400_FAST = "DNBSEQ-G400 FAST"
+    DNBSEQ_G50 = "DNBSEQ-G50"
+    NOT_APPLICABLE = "Not Applicable"
+    NOT_COLLECTED = "Not Collected"
+    NOT_PROVIDED = "Not Provided"
+    MISSING = "Missing"
+    RESTRICTED_ACCESS = "Restricted Access"
+
+
+# Create the enumeration table
+# Pass your enum class and the SQLAlchemy declarative base to enumtables.EnumTable
+_SequencingInstrumentTypeTable = enumtables.EnumTable(
+    SequencingInstrumentType,
+    base,
+    tablename="sequencing_instrument_types",
+)
+
+
+class SequencingProtocolType(enum.Enum):
+    ARTIC_V3 = "artic_v3"
+
+
+# Create the enumeration table
+# Pass your enum class and the SQLAlchemy declarative base to enumtables.EnumTable
+_SequencingProtocolTypeTable = enumtables.EnumTable(
+    SequencingProtocolType,
+    base,
+    tablename="sequencing_protocol_types",
+)
+
+
+class SequencingReads(Entity):
+    __tablename__ = "sequencing_reads"
+    __table_args__ = (UniqueConstraint("s3_bucket", "s3_key"),)
+    __mapper_args__ = {"polymorphic_identity": EntityType.SEQUENCING_READS}
+
+    entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
+    physical_sample_id = Column(Integer, ForeignKey(PhysicalSample.id), nullable=False)
+    physical_sample = relationship(PhysicalSample, backref=backref("sequencing_reads"))
+
+    sequencing_instrument = Column(
+        Enum(SequencingInstrumentType),
+        ForeignKey(_SequencingInstrumentTypeTable.item_id),
+        nullable=False,
+    )
+    sequencing_protocol = Column(
+        Enum(SequencingProtocolType),
+        ForeignKey(_SequencingProtocolTypeTable.item_id),
+        nullable=False,
+    )
+
+    s3_bucket = Column(String, nullable=False)
+    s3_key = Column(String, nullable=False)
+
+    sequencing_date = Column(DateTime)
+
+
+class PathogenGenome(Entity):
+    __tablename__ = "pathogen_genomes"
+
+    entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
+    sequence = Column(String, nullable=False)
+
+    # statistics for the pathogen genome
+    num_unambiguous_sites = Column(Integer, nullable=False)
+    num_n = Column(Integer, nullable=False)
+    num_mixed = Column(Integer, nullable=False)
```

Comment:
it would be nice to find more descriptive names for these fields, I know the bioinformatics people like these names but for me i always have to mentally try to recall what these statistics are referring to. Not saying i want you to change this, just noting.